### PR TITLE
fix: make chat button click event work fine

### PR DESCRIPTION
**Pull request by darkskygit**
fix PD-1504

### CODE REVIEW

**Comment by darkskygit** [comment]
* **#7658** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/7658?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @darkskygit and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/frontend/core/src/components/workspace/index.tsx [hunk 1]
```diff
@@ -72,17 +73,25 @@ export const MainContainer = forwardRef<
...
+  const inChatPanel = useLiveData(
+    activeView.activeSidebarTab$.map(t => t?.id === 'chat')
+  );
   const docId = useLiveData(
     useService(GlobalContextService).globalContext.docId.$
   );
   const docRecordList = useService(DocsService).list;
   const doc = useLiveData(docId ? docRecordList.doc$(docId) : undefined);
   const inTrash = useLiveData(doc?.meta$)?.trash;
+  console.log('ToolContainer', isSidePanelOpen, inChatPanel);
```
Comment (pengx17) [inline packages/frontend/core/src/components/workspace/index.tsx]: remove log

File: packages/frontend/core/src/components/workspace/index.tsx [hunk 2]
```diff
@@ -83,6 +90,7 @@ export const ToolContainer = (props: PropsWithChildren): ReactElement => {
       className={clsx(toolStyle, {
         trash: inTrash,
       })}
+      aria-disabled={isSidePanelOpen && inChatPanel}
```
Comment (EYHN) [inline packages/frontend/core/src/components/workspace/index.tsx]: Why add it here? The ai island has been controlled and hidden in another place.

https://github.com/toeverything/AFFiNE/blob/c7217ed443a86f02161cd0e621144928fae87270/packages/frontend/core/src/components/pure/ai-island/index.tsx#L34

Comment (darkskygit) [reply packages/frontend/core/src/components/workspace/index.tsx]: image

move the island wrapper is not enough, it's parent still blocking events

Comment (darkskygit) [reply packages/frontend/core/src/components/workspace/index.tsx]: i have move the style to ai island

**Review by pengx17 — APPROVED** [review]

**Pull request merged**